### PR TITLE
DEV-1402 Add the new endpoint for Statistic to get Aggregated CO2 over time

### PR DESCRIPTION
**Pull request by AkbarKazimov8**

### CODE REVIEW

**Review by DanielGronau — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version=v3-2023-08-02
+version=v3-2023-08-16
 
 npx redoc-cli@0.13.21 build openapi.yaml --options=theme.json --disableGoogleFont -t ecolytiq.hbs --output index.html
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply build.sh]: done

File: openapi.yaml [hunk 2]
```diff
@@ -2464,6 +2464,104 @@ paths:
...
 
+  /statistic/v1/accounts/{account_id}/footprints:
+    get:
+      operationId: getFootprintsOverTime
+      security:
+        - BearerAuth: [ ]
+      summary: Get total or monthly footprints over time
+      description: Returns an account's total or monthly CO2 footprints over time.
+      tags:
+        - "Statistic API"
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply openapi.yaml:2473]: done

File: openapi.yaml [hunk 3]
```diff
@@ -2464,6 +2464,104 @@ paths:
...
+        - name: sub-category
+          in: query
+          required: false
+          description: Filter by a given sub category.
+          schema:
+            type: string
+          example: accounting
+      responses:
+        '200':
+          description: Footprints for the given account. If the `account_id` doesn't exist, an empty list will be returned.
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply openapi.yaml]: done

File: openapi.yaml [hunk 4]
```diff
@@ -2464,6 +2464,104 @@ paths:
...
+        - BearerAuth: [ ]
+      summary: Get total or monthly footprints over time
+      description: Returns an account's total or monthly CO2 footprints over time.
+      tags:
+        - "Statistic API"
+      parameters:
+        - name: account_id
+          in: path
+          required: true
+          description: The ID of the user/bank account.
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply openapi.yaml]: done

File: openapi.yaml [hunk 5]
```diff
@@ -2464,6 +2464,104 @@ paths:
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiError'
+              examples:
+                Example:
+                  $ref: '#/components/examples/BadRequestGetAggregatedCo2ByMonth-Example'
+        '401':
+          description: Unauthorized. Access token is invalid or expired.
+        '404':
+          description: Not found. The `meta-category` | `sub-category` could not be found.
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply openapi.yaml]: done

File: openapi.yaml [hunk 6]
```diff
@@ -3336,15 +3449,64 @@ components:
...
         error: Bad Request
-        # path: 
+        # path:
 
+    GetTotalFootprintExample:
+      value:
+        intervals:
+          - co2_footprint_in_g: 8793376
+            year: null
+            month: null
```
[the remark this answers is not in the document]
Comment (AkbarKazimov8) [reply openapi.yaml:3472]: done

**Pull request merged**